bin/http_server_spec

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -ex
+gem install bundler
```

Comment:
what ruby version?

Reply:
Based on the Travis CI job logs, it looks like Travis uses ruby 2.4.1.

Reply:
I'm getting this error, but it could be my ruby setup. 
`rbenv: version `2.5' is not installed (set by /Users/andrew/tmp/travis/http-server/bin/http-server-spec/.ruby-version)
`

Reply:
that means you need to use rbenv to install ruby 2.5